vms/vm-under-test/scripts/customize-vm

```diff
@@ -26,4 +26,4 @@ systemctl disable irqbalance
 dnf --enablerepo=rt install -y tuned-profiles-realtime
 dnf --enablerepo=nfv install -y tuned-profiles-nfv-guest
 
-grubby --args="default_hugepagesz=1GB hugepagesz=1G hugepages=1" --update-kernel=ALL
+grubby --args="iommu=pt intel_iommu=on default_hugepagesz=1GB hugepagesz=1G hugepages=1" --update-kernel=ALL
```

Comment:
@matosatti is this setting necessary in our case?

Reply:
After an offline meeting, we have decided to drop it for now.

Reply:
DONE